### PR TITLE
Change DB schema

### LINKED ISSUE #3: Form
# Problem

We would like to allow users to add their conference details in a beautiful form.

# Proposed solution 

Add a form which allows users to:
- select the conference from a drop-down
- enter the code
- add their name
- add their home address (the form should do a geo lookup to find the lat/long.
- add a map of the resulting lat/long, which users can adjust.
- add a departure date.
- optionally add a gravitar, otherwise, add a random custom picture

### PR DESCRIPTION
**Pull request by euclidian**
I think the admin page for Visit model will also fix #3. Please review @meomancer @dimasciput

### CODE REVIEW

File: django_project/core/settings/prod_docker.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import os
 print os.environ
 
-DEBUG = False
+DEBUG = True
```
Comment (dimasciput) [inline django_project/core/settings/prod_docker.py]: Don't put debut true in production

Comment (euclidian) [reply django_project/core/settings/prod_docker.py]: sorry, my bad :)

**Comment by euclidian** [comment]
API returns 

```
[
  {
    "id": 1,
    "name": "Visit",
    "gravatar_url": "https://www.gravatar.com/avatar/d41d8cd98f00b204e9800998ecf8427e",
    "conference_name": "[conf]conf code",
    "home": null,
    "date_left": "2017-02-06T07:06:09Z",
    "gravatar": "",
    "conference": 1
  }
]
```

**Comment by euclidian** [comment]
image

Can't we put this page as user page? I think this quite similiar with @cchristelis request on #3

File: django_project/pycon/models/__init__.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 from .conference import *
 from .visit import *
-from .conference_visit_entry import ConferenceVisitEntry
```
Comment (cchristelis) [inline django_project/pycon/models/__init__.py:6]: rather than comment out, just delete

**Review by cchristelis — COMMENTED** [review]
LGTM

**Comment by euclidian** [comment]
image

I'm sorry, this is the best I can do for UI. :smile: I'm handing the rest to @dimasciput . I spent like 1 hour just to make a datetimeinput and still failed. Good luck dim.

PS : it works though. I can see that on admin page

**Pull request merged**